src/python/pants/util/docutil.py

```diff
@@ -16,8 +16,23 @@ def terminal_width(*, fallback: int = 96, padding: int = 2) -> int:
     return shutil.get_terminal_size(fallback=(fallback, 24)).columns - padding
 
 
-def doc_url(slug: str) -> str:
-    return f"https://www.pantsbuild.org/v{MAJOR_MINOR}/docs/{slug}"
+_VERSIONED_PREFICES = ("docs/", "reference/")
```

Comment:
Prefice or prefix?

Reply:
Oh, apparently prefix -> prefixes, not like appendix -> appendices. Oops.

Reply:
English be cray

Reply:
prefice not being a word in English at all??